### ISSUE TITLE
Is there any way to get goldendict remove the verb tenses characters and "'s" from plural nouns?

### ISSUE THREAD

**Issue by lefthairynut** [goldendict/goldendict#1357]
Hello. I'm getting tired of removing _ed_ from verbs and _'s_ from plural nouns. Is there any way (ANY way) to this trick without manual intervention? I remember _babylon_ did this automatically in windows.

**Comment by vedgy** [comment]
Yes, there is: Morphology/Hunspell. See #528.

**Comment by lefthairynut** [comment]
I managed to add [this](https://www.softmaker.net/down/hunspell/softmaker-hunspell-english-us-101.sox) hunspell dictionary from this [page](https://www.softmaker.com/en/dictionaries) and I have English to English and English to Persian dictionaries but still when I scan the word _books_  via scan popup, it still shows the word _books_ and not _book_ in the program :(
Can you help me with this?

**Comment by vedgy** [comment]
Have you checked the *English (US) Morphology* checkbox on GoldenDict's *Dictionaries->Sources->Morphology* tab? On GNU/Linux the system *hunspell* package should work - there is no need to download dictionaries manually. See also the first reply to #528.

**Comment by lefthairynut** [comment]
there is no dictionary under Available morphology dictionaries. I don't know if it's related or not but I'm running goldendict in portable mode.

**Comment by vedgy** [comment]
Edit your `hunspell dictionariesPath` in *~/.goldendict/config* (or wherever your portable config is) to point to your hunspell dictionaries. For example, mine is:
```xml
<hunspell dictionariesPath="/usr/share/hunspell">
```

There are several *.aff* and *.dic* files in my */usr/share/hunspell*. English dictionaries are supplied by [_hunspell-en* Arch packages_](https://archlinux.org/packages/?sort=&q=hunspell%2Den).

**Comment by lefthairynut** [comment]
I downloaded the package from aur. The setting to change the path of morphology dictionary in the program was disabled anddd when I changed the config file manually as you suggested the program would go back to the default after lunching. in the program it showed that `/bin/content/morphology` was the path it was looking for the dictionary so I ended creating a symlink from the path dictionaries were installed to it. `sudo ln -s /usr/share/hunspell /bin/content/morphology` just delete the _morphology_ directory before.
thanks for the support. much love!
